sources/Load.php

```diff
@@ -2488,44 +2488,29 @@ function determineAvatar($profile)
 function detectServer()
 {
 	global $context;
+	static $server = null;
 
-	$context['server'] = array(
-		'is_iis'			=> serverIs('iis'),
-		'is_apache'			=> serverIs('apache'),
-		'is_litespeed' 		=> serverIs('litespeed'),
-		'is_lighttpd' 		=> serverIs('lighttpd'),
-		'is_nginx' 			=> serverIs('nginx'),
-		'is_cgi' 			=> serverIs('cgi'),
-		'is_windows' 		=> serverIs('windows'),
-		'iso_case_folding' 	=> serverIs('iso_case_folding'),
+	if ($server === null)
+	{
+		$server = new Server($_SERVER);
```

Comment:
Fanciness? O:-)
Actually a piece is missing (i.e. check if the variable is set or not).
```php
if (isset($_SERVER))
    $server = new Server($_SERVER);
else
    $server = new Server(array());
```

I guess I can use $_SERVER by default and in case let it accept an argument to override or nothing if $_SERVER doesn't exist?
/me likes having overrides when possible. :angel: